inc/bench.h

```diff
@@ -58,9 +58,12 @@
 #include <time.h>
 #include <unistd.h>
 #include <wordexp.h>
+
+// temporary flag for 3.0 development TODO need to remove in future
+#define ALLOW_FORBID_FUNC
```

Comment:
should we use this macro to conditionally include "taosdef.h" and "taoserror.h" without just removing them